src/rating/rating.ts

```diff
@@ -28,6 +40,7 @@ export interface StarTemplateContext {
 @Component({
   selector: 'ngb-rating',
   changeDetection: ChangeDetectionStrategy.OnPush,
+  host: {'(keydown)': 'handleKeyDown($event)'},
```

Comment:
Not sure if should be 
- `keyup` - only increments by 1 once you release the key
- `keydown` - increments continuously by 1 while you hold the key